air/src/execution_step/errors/uncatchable_errors.rs

```diff
@@ -97,6 +97,17 @@ pub enum UncatchableError {
 
     #[error("failed to deserialize to CallServiceFailed: {0}")]
     MalformedCallServiceFailed(serde_json::Error),
+
+    /// Argument hash or tetraplet mismatch in a call merged from current_data
+    #[error("call's {param} doesn't match expected parameters: expected {expected_value}, got {current_value} ")]
+    CallParametersMismatch {
```

Comment:
Could you elaborate if the word Parameter in CallParametersMismatch should be plural or singular here?